xen-bugtool

```diff
@@ -1516,6 +1521,31 @@ def multipathd_topology(cap):
 
     return stdout
 
+def filter_snmp_xs_conf(_):
+    """Filter /etc/snmp/snmp.xs.conf with keys and community removed"""
+    try:
+        with open("/etc/snmp/snmp.xs.conf", "r") as file:
```

Comment:
Similar to `SYSCONFIG_SNMPD`, declare file or folder at file beginning.

Reply:
Updated.